repository_service_tuf/cli/admin/metadata.py

```diff
@@ -0,0 +1,605 @@
+
+
+def _keys_removal(current_root: RootInfo):
+    """Asking the user if he wants to remove any of the root keys"""
+    while True:
+        keys_table = _create_keys_table(current_root.keys, current_root, False)
+        console.print("Here are the current root keys:")
+        console.print(keys_table)
+        console.print("\n")
+
+        key_removal = prompt.Confirm.ask("Do you want to remove a key")
+        if not key_removal:
+            break
+
+        name = prompt.Prompt.ask(
+            "[green]Name/Tag/ID prefix[/] of the key to remove"
+        )
+        if not current_root.remove_key(name):
+            console.print(
+                "\n", f":cross_mark: [red]Failed[/]: key {name} is not in root"
+            )
+            continue
+        else:
+            console.print(f"Key with name/tag [yellow]{name}[/] removed\n")
+            if len(current_root.keys) < 1:
+                console.print("No keys are left for removal.")
+                break
+
+
+def _keys_additions(current_root: RootInfo):
+    root_threshold = current_root.threshold
+    console.print(
+        f"You need to have at least [cyan]{root_threshold}[/] signing keys."
+    )
+    while True:
+        signing_list = [
+            key.to_dict() for key in current_root.signing_keys.values()
+        ]
+        keys_table = _create_keys_table(signing_list, current_root, False)
+        console.print("\nHere are the current root signing keys:")
+        console.print(keys_table)
+        keys_amount = len(current_root.signing_keys)
+        if root_threshold <= keys_amount:
+            agree = prompt.Confirm.ask("\nDo you want to add a new key?")
+            if not agree:
+                return
+        else:
+            remaining = root_threshold - keys_amount
+            console.print(f"You must add {remaining} more signing key(s)")
+
+        root_key: RSTUFKey = _get_key(Root.type)
+        if root_key.error:
+            console.print(f":cross_mark: [red]Failed[/]: {root_key.error}")
+            continue
+
+        if root_key == current_root.online_key:
+            console.print(
+                ":cross_mark: [red]Failed[/]: This is the current online key. "
+                "Cannot be added"
+            )
+            continue
+
+        if current_root.is_keyid_used(root_key.key["keyid"]):
+            console.print(":cross_mark: [red]Failed[/]: Key is already used")
+            continue
+
+        root_key.name = prompt.Prompt.ask(
+            "[Optional] Give a [green]name/tag[/] to the key"
+        )
+
+        current_root.add_key(root_key)
+
+
+def _get_positive_int_input(msg: str, input_name: str, default: Any) -> int:
+    input: int = 0
+    while True:
+        input = prompt.IntPrompt.ask(msg, default=default, show_default=True)
+        if input < 1:
+            console.print(f"{input_name} must be at least 1")
+        else:
+            break
+
+    return input
```

Comment:
nit: this could be
```suggestion
        if input >= 1:
            return input

        console.print(f"{input_name} must be at least 1")

```